Code/Framework/AzCore/AzCore/Script/ScriptContext.cpp

```diff
@@ -3408,7 +3408,14 @@ LUA_API const Node* lua_getDummyNode()
                     const BehaviorParameter* arg = method->GetArgument(iArg);
                     BehaviorClass* argClass = nullptr;
                     LuaLoadFromStack fromStack = FromLuaStack(context, arg, argClass);
-                    AZ_Assert(fromStack, "Argument %s for Method %s doesn't have support to be converted to Lua!", arg->m_name, method->m_name.c_str());
+                    AZ_Assert(fromStack,
+                        "The argument type: %s for method: %s is not serialized and/or reflected for scripting.\n"
+                        "Make sure %s is added to the serialization context and reflected to the Behavior Context\n"
```

Comment:
nit: why is serialization context not capitalized, but behavior context is? :)

Reply:
No reason in particular, I'll update it :)